### PR TITLE
Add some logging to molotov

### PR DESCRIPTION
**Pull request by optimumtact**
Who primed molotov and the area of priming

also messages the admins

### CODE REVIEW

**Comment by freerealestate** [comment]
why this but not a nerf

**Comment by optimumtact** [comment]
http://www.xeon.pw/~francis/memes/webm/its_only_game.webm

**Comment by freerealestate** [comment]
i've never seen a molotov in-game, so just wondering

it seems like instead of fixing the problem you're just pointing it out, or something

File: code/modules/food&drinks/drinks/drinks/bottle.dm [hunk 1]
```diff
@@ -300,6 +300,11 @@
 /obj/item/weapon/reagent_containers/food/drinks/bottle/molotov/attackby(obj/item/I, mob/user, params)
 	if(is_hot(I) && !active)
 		active = 1
+		var/turf/bombturf = get_turf(src)
+		var/area/bombarea = get_area(bombturf)
+		message_admins("[key_name(user)]<A HREF='?_src_=holder;adminmoreinfo=\ref[user]'>?</A> has primed a [name] for detonation at <A HREF='?_src_=holder;adminplayerobservecoodjump=1;X=[bombturf.x];Y=[bombturf.y];Z=[bombturf.z]'>[bombarea.name] (JMP)</a>.")
+		log_game("[key_name(user)] has primed a [name] for detonation at [bombarea.name] ([bombturf.x],[bombturf.y],[bombturf.z]).")
+
```
Comment (MrStonedOne) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: What is A.name

Should be get_area().name you don't even need to do name, as byond converts reference types in [] to a string containing its name. so [get_area(src)] works.

**Comment by optimumtact** [comment]
What problem? They lack admin logging when they should have it. What other problem is there?

File: code/modules/food&drinks/drinks/drinks/bottle.dm (hunk 1 above)
Comment (MrStonedOne) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: you don't even need to do name, as byond converts reference types in [] to a string containing its name. so [get_area(bombturf)] works.

Comment (freerealestate) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: I read people saying they're very OP in IRC.

Comment (freerealestate) [inline code/modules/food&drinks/drinks/drinks/bottle.dm]: I assumed this power was why you were adding logging.

**Comment by SconesC** [comment]
Thanks orange man

**Comment by Ergovisavi** [comment]
I don't see why not, and I also don't see why?

This is already under combat logs, unless I'm mistaken, hitting someone with a molotov should get logged as X hits Y with the molotov.

**Comment by SconesC** [comment]
Considering what can fucking happen with /certain/ contents they need the same logging as grenades

**Comment by optimumtact** [comment]
knowing who primed a molotov and where is also useful information

**Comment by freerealestate** [comment]
Yeah, like, about what scones said. The logging is necessitated by their
ridiculousness, but rather than fix that you're just adding logs to it.
That's kind of weird.


posted from my durnkphone

**Comment by optimumtact** [comment]
dude I dont' care about how powerful they are, even if they were less powerful they still need this information.

I tend to avoid making balance changes

**Pull request merged**